examples/mysql-wordpress-pd/README.md

```diff
@@ -336,27 +266,33 @@ http://<external-ip>
 
 You should see the familiar WordPress init page.
 
+![WordPress init page](WordPress.png "WordPress init page")
+
 ## Take down and restart your blog
 
-Set up your WordPress blog and play around with it a bit.  Then, take down its pods and bring them back up again. Because you used persistent disks, your blog state will be preserved.
+Set up your WordPress blog and play around with it a bit. Then, take
+down its pods and bring them back up again. Because you used
+persistent disks, your blog state will be preserved.
 
-If you are just experimenting, you can take down and bring up only the pods:
+All of the resources are labeled with `app=wordpress`, so you can
+easily bring them down with:
 
-```sh
-$ kubectl delete -f examples/mysql-wordpress-pd/wordpress.yaml
-$ kubectl delete -f examples/mysql-wordpress-pd/mysql.yaml
+```shell
+kubectl delete deployment,pod,service -l app=wordpress
 ```
 
-When you restart the pods again (using the `create` operation as described above), their services will pick up the new pods based on their labels.
+Later, re-creating the resources with the original commands will pick
+up the original disks with all your data intact. We did not delete the
+PV Claims. If we did that, our disks might be claimed by other pods in
```

Comment:
nit: This might be easier to read as "Because we did not delete the PV Claims, no other pods in the cluster could claim them after we deleted our pods.  Keeping the PV Claims also ensured recreating the Pods did not cause the PD to switch Pods."